index.html

```diff
@@ -14,15 +14,21 @@
 <body>
         <header class="header">
             <nav class="nav">
-                <h1><a href="">Kigozi Joseph</a></h1>
+                <h1><a href="https://github.com/zeph180" ref="noreferrer">Kigozi Joseph</a></h1>
                 <i class="fas fa-bars fa-lg"></i>
+                <ul>
```

Comment:
- [ ] As the previous reviewer noted, I would also recommend you make this hamburger icon to display none on the desktop version.
![image](https://user-images.githubusercontent.com/86200356/206877145-83b95de1-b146-4308-8cc6-3f918f0d20ae.png)